framework/src/main/java/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -432,6 +453,24 @@ protected void addMainToArgs(final List<String> args) {
             args.add(quote(concatenatePaths(ppOpts)));
         }
 
+        // Get CHECKERFRAMEWORK environment variable
+        final String CHECKERFRAMEWORK = System.getenv("CHECKERFRAMEWORK");
+
+        // addding "--patch-module" arguments to compiler Arguments
+        for (final String argLine : expandArgFiles(argListFiles)) {
```

Comment:
Does this really need to go through all argfiles?
Isn't this substitution only meaningful for the `Patch_Modules_argfile` file?
Why not define your own simpler format for that file and here you write code that makes the relative paths absolute?